client/signup/main.jsx

```diff
@@ -249,13 +249,7 @@ class Signup extends React.Component {
 
 		const p2SiteStep = this.props.progress[ 'p2-site' ];
 
-		if (
```

Comment:
This is just my hack from last week (https://github.com/Automattic/wp-calypso/pull/44172) which is kinda hacky so I had to update it here to make it work again. It's only for P2 signup flow and I'll be probably updating it soon in https://github.com/Automattic/wp-calypso/pull/43522 once this PR lands.